src/ktopt_planning_context.cpp

```diff
@@ -100,7 +173,16 @@ void KTOptPlanningContext::setRobotDescription(std::string robot_description)
 
   std::tie(plant_, scene_graph_) = AddMultibodyPlantSceneGraph(&builder, 0.0);
 
-  auto robot_instance = Parser(plant_, scene_graph_).AddModels(robot_description_);
+  // TODO:(kamiradi) change to the following once you get a proper description
+  // frm the manager
+  // auto robot_instance = Parser(plant_, scene_graph_).AddModelsFromString(robot_description_);
+
+  // HACK: For now loading directly from drake's package map
+  const char* ModelUrl = 
```

Comment:
Put even hacky constants into an anonymous namespace at the top of the file to make sure they won't be overlooked :sweat_smile: